<h3 style="color: #001a79;">Exercise 4</h3>

<hr style="border-top: 1px solid #001a79;" />

Note that there are the same number of ways to get 4 tails as there to get 4 heads. Explain why this is.

The general formula is: $${n \choose r} = {n! \over r!(n-r)!}$$

where n is the number of things to choose from, and we choose r of them. 

https://www.mathsisfun.com/combinatorics/combinations-permutations.html

n is the number of coins and r is the number of tails. 

In [2]:
r = 4
n = 10

In [5]:
import math
combinations = (math.factorial(n)) // ((math.factorial(r)) * (math.factorial(n - r)))
combinations

210

This returns the same number of combinations if we set r to be the number of heads. 

It doesnt matter whether it is (4 heads and 6 tails) or (4 tails and 6 heads) the number of combinations will be 210.  